locale/en/get-involved/node-speakers.md

```diff
@@ -112,6 +112,16 @@ If anyone is reported as malicious or making others feel uncomfortable, they may
 - [LinkedIn](https://linkedin.com/in/tracyslee)
 - Topics - Node.js Community, Women in Tech, Diversity & Inclusion, RxJS & Node.js
 
+#### Florida 🌴
```

Comment:
Could you please remove the icon just for consistency?
```suggestion
#### Florida
```

Reply:
Fixed! Sorry silly habit :)